Marlin/src/MarlinCore.cpp

```diff
@@ -533,6 +533,20 @@ inline void manage_inactivity(const bool ignore_stepper_queue=false) {
     }
   #endif
 
+  #if HAS_FILAMENTCHANGE
```

Comment:
`HAS_FILAMENTCHANGE` is not defined.

As a general principle, please complete and test your feature before submitting the PR.
This saves us the time required to type comments like this one.

Reply:
HAS_FILAMENTCHANGE is defined.

I tested it and it works.